xcodeproj/internal/xcode_targets.bzl

```diff
@@ -513,41 +513,51 @@ _PREVIEWS_ENABLED_PRODUCT_TYPES = {
 
 def _set_swift_include_paths(
         *,
+        build_mode,
         build_settings,
         xcode_generated_paths,
         xcode_target):
     if not xcode_target.swift_params:
         return
 
-    def _handle_swiftmodule_path(file):
-        path = file.path
-        bs_path = xcode_generated_paths.get(path)
-        if bs_path:
-            include_path = paths.dirname(bs_path)
-        else:
+    if build_mode == "xcode":
+        def _handle_swiftmodule_path(file):
+            path = file.path
+            bs_path = xcode_generated_paths.get(path)
+            if bs_path:
+                include_path = paths.dirname(bs_path)
+            else:
+                include_path = file.dirname
+
+            if include_path.find(" ") != -1:
+                include_path = '"{}"'.format(include_path)
```

Comment:
Why not always `.format`? Is this a perf related concern?

Reply:
I'm not changing behavior here. But it's not wrapping in quotes when not needed to keep the size of the generated project down.

Reply:
Just noted similar code that existed previously as well, my bad as I thought this was net new. Regardless, got it, not change in behaviour and it makes sense to keep proj size down. Thx for clarifying!